contracts/mocks/MockFarmBot.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.0;
+
+import "../ubeswap/contracts/uniswapv2/interfaces/IUniswapV2Router02.sol";
+import "../ubeswap/contracts/uniswapv2/interfaces/IUniswapV2Router02SwapOnly.sol";
+import "./MockLPToken.sol";
+
+/**
+* MockFarmBot has the bare minimum functionality of a farm bot for testing RevoFPBroker.
+*/
+contract MockFarmBot is MockERC20 {
+    IUniswapV2Router02SwapOnly public swapRouter; // address to use for router that handles swaps
```

Comment:
nit: `swapRouter` isn't actually necessary for the broker contract, right?

Reply:
an address is given in the constructor so I went ahead and saved it